docs/cspell.json

```diff
@@ -0,0 +1,699 @@
+{
+    // Version of the setting file.  Always 0.2
+    "version": "0.2",
+    // language - current active spelling language
+    "language": "en",
+    // words - list of words to be always considered correct
+    "words": [
+"ANPAW",
+"AWSARN",
+"AWSIIDTTL",
+"iamserviceaccount",
+"IRSA",
+"RPID",
+"subkind",
+"sudoer",
+"unconfigured",
+"unprefixed",
+"vmcopy",
+"webclient",
+"sharded",
+"keymap",
+"APPDATA",
+"killall",
+"LOCALAPPDATA",
+"loginrule",
+"etcdctl",
+"plugindata",
+"Println",
+"Sprintf",
+"watcherjob",
+"pseudoversion",
+"allowedlogins",
+"attrname",
+"datadoghq",
+"flexi",
+"idps",
+"myns",
+"prioritise",
+"SAMLIDP",
+"samltest",
+"samltestid",
+"scrollback",
+"vkxz",
+"XNCK",
+"cond",
+"FGHIJ",
+"replaceall",
+"anotheruser",
+"auths",
+"CHANGEME",
+"cimg",
+"circleci",
+"CLOUDSDK",
+"clusterolebinding",
+"deregisters",
+"devicetrust",
+"gsutil",
+"IAMR",
+"initcontainers",
+"mermaidjs",
+"netcat",
+"nosql",
+"obtainlicense",
+"octocat",
+"pgaadauth",
+"rdsproxy",
+"rollouts",
+"rtrzn",
+"uninstallation",
+"workgroups",
+"yourdatabase",
+"ioreg",
+"ndots",
+"GHES",
+"backoff",
+"NOKEY",
+"awsapp",
+"acpi",
+"clis",
+"httpout",
+"localca",
+"myproject",
+"MYTOKEN",
+"navin",
+"nvme",
+"priya",
+"splunkd",
+"splunkforwarder",
+"tcpout",
+"timechart",
+"westeurope",
+"splunkd",
+"Traefik",
+"xeyes",
+"dockerized",
+"cloudkms",
+"mycluster",
+"myrolebinding",
+"Rdik",
+"serviceaccountname",
+"storageenabled",
+"xlink",
+"actionstest",
+"auditctl",
+"auditd",
+"ausearch",
+"authserver",
+"cqlsh",
+"datareader",
+"keyfile",
+"Keyspaces",
+"loginuid",
+"managedclusters",
+"mypartial",
+"roadmap",
+"rolearn",
+"runcommand",
+"selfsubjectrulesreviews",
+"tokenconfig",
+"truststore",
+"auditctl",
+"auditd",
+"authserver",
+"AADUSER",
+"ABCDEFGHIJKL",
+"abcdefghijklm",
+"aclfile",
+"adcli",
+"Addrs",
+"ADFS",
+"adutil",
+"Afax",
+"agentless",
+"agentpool",
+"AICPA",
+"AICPA’s",
+"AKIA",
+"allcommands",
+"allkeys",
+"allowdeny",
+"AMAZ",
+"apikey",
+"apiserver",
+"appdomain",
+"APPSECRET",
+"appuser",
+"AQAB",
+"Aqxs",
+"Archlinux",
+"armv",
+"atburke",
+"authc",
+"AUTHINFO",
+"authkey",
+"authservers",
+"Authy",
+"authz",
+"automount",
+"autoscale",
+"awscli",
+"awsconsole",
+"awsdatabases",
+"awsuser",
+"azuread",
+"azuredatabases",
+"backporting",
+"backports",
+"BDRVJUSUZ",
+"behaviour",
+"bfhunter",
+"Binm",
+"bjoerger",
+"bluemix",
+"boto",
+"BSFD",
+"BUCKETNAME",
+"buildbox",
+"caddef",
+"categorisation",
+"cavium",
+"CCDC",
+"centralus",
+"certificatekey",
+"certutil",
+"cfsdf",
+"Cgajq",
+"cgroupv",
+"chacha",
+"cicd",
+"ciphersuites",
+"clientid",
+"cloudbuild",
+"cloudsql",
+"clusers",
+"clusteradmin",
+"clustername",
+"clusterrole",
+"clusterrolebinding",
+"cockroachdb",
+"codingllama",
+"configmap",
+"connectorname",
+"createkey",
+"creds",
+"crond",
+"CTAP",
+"customizability",
+"journalctl",
+"datacenter",
+"dbaccessdemo",
+"dbadir",
+"dbadmin",
+"dbeaver",
+"dbgroup",
+"dbname",
+"DBSIZE",
+"dbuser",
+"DEBU",
```

Comment:
Probably a real mispelling, should be `DEBUG`

Reply:
Similar to ERRO we have output that has this